faqs/stocks/which-screener.md

```diff
@@ -9,75 +9,122 @@ description: >-
 
 ## Introduction
 
-Even though screeners are useful in finding out the investment opportunities which suits your criteria and to monitor the existing investment, it might not be a good idea to take a screener at its face value.
+Though stock-screeners calculate a lot of things, you must do you own calculations too. You will miss a lot of good opportunities or make some expensive mistakes if you consider only the pre-calculated values.
 
-You might see an opportunity or a red-flag from these screeners. However, it could easily just be a calculation error, or usage of incorrect formula, or not removing an outlier.
+The objective of this article is not to demean the websites but to show the importance of your own calculations.
 
-This article is to sensitize you that you might have to do more research rather than just going with the presented data from the UI of a screener.
+----
 
-## Examples of incorrect data in screeners
+## Examples of dividend yield
 
-We've come across some incorrect computations and results that are outright wrong. We've also informed the teams behind these screeners. At the time of writing this, these have not been addressed.  
-  
-It's our duty to warn others to be mindful of these, while they embark on their research.
+Lets consider the dividend yield of Britannia Industries as on 29th April 2021:
 
-Following are a few examples of such issues we'd noticed when we were doing this research into existing screeners.
+```
+Screener.in:    4.18%
+MoneyControl:   0.95%
+MorningStar.in: 1.78%
 
-### Gross Profit Margin
+Correct value:  ~2-5%
+Depends on your judgement on how repeatable these dividends are
+```
 
-The Gross Profit Margin on screener.in for all IT companies, including Tata Consultancy Services \(TCS\), is either 100%, or close to 100%.
+The differences are because of different treatments of interim dividends. While `screener.in` considers all the interim dividends of FY21, the `MoneyControl` considers only FY20's dividends. `MorningStar` considered only one of the two interim dividends (probably assuming the other one as a one time special dividend).
 
-![Gross Profit Margin of IT companies on screener.in - Dark Mode](../../.gitbook/assets/screener-gpm-dark.png)
 
-![Gross Profit Margin of IT companies on screener.in - Light Mode](../../.gitbook/assets/screener-gpm-light.png)
+Another interesting case is of Majesco. The dividend yield as on 29th April 2021 on various websites is:
 
-This basically implies that TCS has nearly ₹0 worth of expenses and all their sales is translated into gross profit.
+```
+Screener.in:              1,353 %
```

Comment:
Why are percentage symbols having one space each, from the number?

We can write it as `1,353%`.